Mancala.hs

```diff
@@ -98,7 +154,7 @@ circular_map l start n f
 
 
 
-move_pieces :: Action -> Board -> Board
+move_pieces :: Action -> Board -> Board --needs to also return the index of the last dropped element to make life easier
```

Comment:
Ya it probably makes more sense return both the updated board and the index in a tuple

Reply:
We can easily implement a second turn and capture then once this is good!